### DISCUSSION TITLE
Is there any way to try it to understand it's capability?

### DISCUSSION THREAD

**Discussion by skt1483** [microsoft/aisdkforsapabap#11] [Q&A] [answered]
Hi Experts,

Do we have any videos'/slides or or a way to explore further to know get in depth understanding of 'aisdkforsapabap".

Is it integrated with GPt-4 ?

Regards,
Saurabh

**Comment by gopalnair** [comment, marked as answer]
Hi Saurabh,
   We will be releasing video tutorials ...etc in the future. However, for now, you can refer to the documentation site at : https://microsoft.github.io/aisdkforsapabap/

In the "Docs" link, there is a section dedicated to "Installing the SDK". In short, use ABAPGit to import the public git repo to your application server.

If you would like to use your own application server, I would recommend going with SAP CAL instance. I recommend ABAP 1909 system. 

Best Regards,
Gopal Nair.
